server/neptune/workflows/activities/github/markdown/templates/checkrun.tmpl

```diff
@@ -14,4 +14,9 @@
 If a specific terraform operation has failed, check the logs (linked above) to determine if this is a terraform error or not.  If it is, you'll likely need to make some configuration fixes and try again.
 
 **Note: If the terraform operations do not indicate any failure, there is a likely a platform issue. Contact service owners for additional debugging.**
+{{end}}
+{{if .TimedOut }}
+## Timeout :clock1:
+:point_right: One more more operations has timed out. It's possible that a terraform operation is still making progress, this can be confirmed by checking the logs above. However,
```

Comment:
nit: 
```suggestion
:point_right: One or more operations has timed out. It's possible that a terraform operation is still making progress, this can be confirmed by checking the logs above. However,
```